src/nvim/version.c

```diff
@@ -1106,7 +1112,8 @@ static void do_intro_line(int row, char_u *mesg, int attr)
         clen += byte2cells(p[l]);
       }
     }
-    screen_puts_len(p, l, row, col, *p == '<' ? hl_attr(HLF_8) : attr);
+    assert(row < INT_MAX && col < INT_MAX);
```

Comment:
screen_puts_len is called in many places in the code base. I think in the future it should be refactored to the *row* and *col* parameters to be *long*, and not *int*.